Plugins/BTCPayServer.RockstarDev.Plugins.Payroll/Data/Models/PayrollUser.cs

```diff
@@ -20,9 +20,17 @@ public class PayrollUser
     public ICollection<PayrollInvoice> PayrollInvoices { get; set; } = new List<PayrollInvoice>();
 
     // TODO: Adding State property
+    //public PayrollUserState State { get; set; }
     // Having a public page where user can visit the link and complete registration by entering password and activating account
 
     internal static void OnModelCreating(ModelBuilder modelBuilder)
     {
     }
 }
+
+public enum PayrollUserState
+{
+    Active,
+    Deactivated,
+    Archive
+}
```

Comment:
I would want:

0 = Disabled
1 = Active
2 = Archived

And then when database migration is created, we should also go with migrating existing data, plus ensuring that users are created in Active state.